pages/login/index.vue

```diff
@@ -27,10 +33,15 @@
             ref="passwordField"
             :options="{ placeholder: 'Password' }"
             :base-style="passwordBaseStyle"
-            class-name="sign-in-password"
+            :class-name="isInvalidPassword ? 'sign-in-password is-invalid' : 'sign-in-password'"
```

Comment:
is-invalid should be added by bootstrap if you set the :state to false

Reply:
We are not using bootstrap on this component as it is a ux component